build.sbt

```diff
@@ -636,10 +638,9 @@ lazy val integrationTests = (project in file("integration-tests"))
       "org.projectlombok" % "lombok" % "1.18.32" % Provided,
       "org.apache.spark" %% "spark-sql" % sparkVersion % Test,
       "io.delta" %% "delta-spark" % deltaVersion % Test,
-      "org.apache.hadoop" % "hadoop-aws" % "3.3.6" % Test,
-      "org.apache.hadoop" % "hadoop-azure" % "3.3.6" % Test,
+      "org.apache.hadoop" % "hadoop-aws" % hadoopVersion % Test,
```

Comment:
Before this PR,  we use the `hadoop 3.3.6`  to run the integration tests by default.  while actually,  I encountered a serious issue here https://github.com/unitycatalog/unitycatalog/issues/1092#issuecomment-3470427971.   

I think I will need a separate PR to address the cross-hadoop version issue.